src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -210,4 +210,7 @@
   <data name="JsonDeserializationFailed" xml:space="preserve">
     <value>Conversion from JSON failed with error: {0}</value>
   </data>
+  <data name="NotSuccessStatusCode" xml:space="preserve">
+    <value>The remote server returned an error: ({0:d}) {1}.</value>
```

Comment:
d is for decimal:) According to this [doc](https://msdn.microsoft.com/en-us/library/a0h36syw(v=vs.110).aspx), the format string can only contain the "G" or "g", "D" or "d", "X" or "x", and "F" or "f", i.e. it can't be unambiguous "n" or "#". As an alternative, I can cast the enum to int explicitly without all these hacks)